Vagrantfile

```diff
@@ -1,83 +1,108 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
-# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
-VAGRANTFILE_API_VERSION = "2"
-
 $script = <<SCRIPT
-GOVERSION="1.7.3"
-SRCROOT="/opt/go"
-SRCPATH="/opt/gopath"
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+GOVERSION='1.7.3'
+
+SRCROOT='/opt/go'
+SRCPATH='/opt/gopath'
+
+umask 022
 
-# Get the ARCH
-ARCH="$(uname -m | sed 's|i686|386|' | sed 's|x86_64|amd64|')"
+ARCH=$(uname -m | sed 's/i686/386/;s/x86_64/amd64/')
 
-# Install Prereq Packages
 export DEBIAN_PRIORITY=critical
 export DEBIAN_FRONTEND=noninteractive
 export DEBCONF_NONINTERACTIVE_SEEN=true
-APT_OPTS="--yes --force-yes --no-install-suggests --no-install-recommends"
+
+APT_OPTIONS='--assume-yes --no-install-suggests --no-install-recommends'
```

Comment:
I agree with moving from --force-yes to --assume-yes.

Could we not simply modify this line to the following to resolve the issue? This would preclude the need for major changes to the rest of the file.
```bash
APT_OPTS="--assume-yes --no-install-suggests --no-install-recommends -o Dpkg::Options::=\"--force-confdef\" -o Dpkg::Options::=\"--force-confold\""
```